data/src/test/java/org/apache/iceberg/data/TestDataFileIndexStatsFilters.java

```diff
@@ -137,9 +135,11 @@ public void testPositionDeletePlanningPath() throws IOException {
       tasks = Lists.newArrayList(tasksIterable);
     }
 
-    Assert.assertEquals("Should produce one task", 1, tasks.size());
+    Assertions.assertThat(tasks).as("Should produce one task").hasSize(1);
     FileScanTask task = tasks.get(0);
-    Assert.assertEquals("Should have one delete file, file_path matches", 1, task.deletes().size());
+    Assertions.assertThat(task.deletes().size())
```

Comment:
should be `Assertions.assertThat(task.deletes()).as(...).hasSize(1)`, because the `.as(..)` needs to come before the final assertion